package/MDAnalysis/core/groups.py

```diff
@@ -2554,19 +2554,46 @@ def write(self, filename=None, file_format="PDB",
             ``"conect"``: write only the CONECT records defined in the original
             file. ``"all"``: write out all bonds, both the original defined and
             those guessed by MDAnalysis. ``None``: do not write out bonds.
-            Default os ``"conect"``.
+            Default is ``"conect"``.
+        frames:
+            An ensemble of frames to write. The ensemble can be an list or
+            array of frame indices, a mask of booleans, an instance of
+            :class:`slice`, or an indexed trajectory. By default, 'frames' is
```

Comment:
"indexed trajectory" is very hard to understand for anyone who hasn't looked at this PR in detail. Ideally we add something to the coordinates docs update #1827 but at least an example in a Example section should be here, especially now that `AtomGroup.write()` has truly become the all-singing-all-dancing-all-writing method.